src/openfe_analysis/utils/lassoo_highlights.py

```diff
@@ -0,0 +1,333 @@
+# This file is thanks Christian W. Feldman
```

Comment:
Can we put plotting into openfe main repo please?

Reply:
This could actually be a dependency, it's vendored code:
https://github.com/c-feldmann/lassohighlight/tree/master